artifactory.py

```diff
@@ -666,15 +667,18 @@ def rest_post(
         """
         Perform a POST request to url with requests.session
         """
-        res = session.post(
+        response = session.post(
             url,
             params=params,
             headers=headers,
             verify=verify,
             cert=cert,
             timeout=timeout,
         )
-        return res.text, res.status_code
+        if response.status_code not in (200, 201):
```

Comment:
`204` is also a valid response in POST requests.

Let's use this one - `response.raise_for_status()`